From e12704bfc7c4192385dc8c5d600020439c0319df Mon Sep 17 00:00:00 2001
From: Willy Tarreau <w@1wt.eu>
Date: Tue, 15 Jul 2014 19:06:18 +0200
Subject: [PATCH 45/53] MINOR: session: export the function
 'smp_fetch_sc_stkctr'

This one is sometimes useful outside of this file.
---
 include/proto/session.h | 1 +
 src/session.c           | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/proto/session.h b/include/proto/session.h
index c835bf0..2389a79 100644
--- a/include/proto/session.h
+++ b/include/proto/session.h
@@ -45,6 +45,7 @@ void session_process_counters(struct session *s);
 void sess_change_server(struct session *sess, struct server *newsrv);
 struct task *process_session(struct task *t);
 void default_srv_error(struct session *s, struct stream_interface *si);
+struct stkctr *smp_fetch_sc_stkctr(struct session *l4, const struct arg *args, const char *kw);
 int parse_track_counters(char **args, int *arg,
 			 int section_type, struct proxy *curpx,
 			 struct track_ctr_prm *prm,
diff --git a/src/session.c b/src/session.c
index df85170..aeaa7e1 100644
--- a/src/session.c
+++ b/src/session.c
@@ -2758,7 +2758,7 @@ void session_shutdown(struct session *session, int why)
  * to be able to convery multiple values per key (eg: have gpc0 from
  * multiple tables).
  */
-static struct stkctr *
+struct stkctr *
 smp_fetch_sc_stkctr(struct session *l4, const struct arg *args, const char *kw)
 {
 	static struct stkctr stkctr;
-- 
1.8.5.5

